src/io/CavityWriter.h

```diff
@@ -5,17 +5,34 @@
 
 #include "plugins/PluginBase.h"
 
+class CavityEnsemble;
+
 class CavityWriter : public PluginBase {
 public:
     CavityWriter(){}
 	CavityWriter(unsigned long writeFrequency, std::string outputPrefix, bool incremental);
-	~CavityWriter();
+	~CavityWriter(){};
 	//! @todo comment
 	
 	void readXML(XMLfileUnits& xmlconfig);
```

Comment:
the documentation from the cpp file should be here

Reply:
Changed as requested c9a26ad90439b66e658c293bae6cb914eba4b29c